css/house-style.md

```diff
@@ -152,14 +152,14 @@ table {
 **Non-js fallbacks/JS only styles**. We do this:
 ```scss
 .c-class {
+	display: block;
+
     .js & {
         display: none;
```

Comment:
Maybe a different enhanced example here? It looks odd that a JS enhancement is actually hiding something. `display: flex` instead?